marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/entities/testTdeGenerationDisabled.sjs

```diff
@@ -0,0 +1,35 @@
+/**
+ Copyright (c) 2020 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+const test = require("/test/test-helper.xqy");
+
+function checkForTde(uri){
+  return fn.head(xdmp.eval(
+    `fn.docAvailable('${uri}') `,
+    {uri:uri}, {database: xdmp.schemaDatabase()}
+  ));
+}
+
+function verifyTdeGeneration() {
+  return [
+    test.assertTrue(checkForTde("/tde/Customer-0.0.1.tdex")),
```

Comment:
Really need assertion messages here to explain why it's true or false for each one.